roles/mongodb/tasks/users.yml

```
@@ -0,0 +1,18 @@
+---
+
+- name: users > Create Mongo users
+  mongodb_user:
+    name: "{{ item.name }}"
+    password: "{{ item.password|default(omit) }}"
+    database: "{{ item.database }}"
+    roles: "{{ item.roles|default(omit) }}"
+    state: "{{ item.state|default(omit) }}"
+    login_database: "{{ item.login_database|default(omit) }}"
+    login_host: "{{ item.login_host|default('localhost') }}"
```

```suggestion
    login_host: "{{ item.login_host|default(omit) }}"
```
lets keep same strategy to handle default values

---

I kept default values from the ansible module, so it's easy to read without searching documentation

---

```diff
--- login_database: "{{ item.login_database|default(omit) }}"
+++ login_database: "{{ item.login_database|default('admin') }}"
```